.github/workflows/release_workspace.yml

```diff
@@ -4,133 +4,39 @@ on:
   workflow_dispatch:
     inputs:
       workspace:
-        description: "Name of the Workspace"
+        description: 'Name of the Workspace'
         required: true
         type: string
-      force_release:
-        description: "Force release even if no changesets are present"
-        required: false
-        type: boolean
-      branch:
-        description: "Branch to run the workflow on"
-        required: false
-        default: "main"
-        type: string
   workflow_call:
     inputs:
-      force_release:
-        description: "Force release even if no changesets are present"
-        required: false
-        type: boolean
       workspace:
-        description: "Name of the Workspace"
+        description: 'Name of the Workspace'
         required: true
         type: string
-      branch:
-        description: "Branch to run the workflow on"
-        required: false
-        default: "main"
-        type: string
 
 concurrency:
   group: ${{ github.workflow }}-${{ inputs.workspace }}
 
 jobs:
-  changesets-pr:
```

Comment:
all of these have been moved to the `create_changeset_pr_for_workspace` workflow